open_discussions/templates/base.html

```diff
@@ -13,6 +13,8 @@
     {% if js_settings_json %}
     <script type="text/javascript">
     var SETTINGS = {{ js_settings_json|safe }};
+    var csrfToken = '{{ csrf_token }}';
```

Comment:
what's this for / is it necessary?

Reply:
That is for making requests to the django backend. It's a django generated token that the backend reads and verifies that it's not a cross-site request forgery. If your project has a different way of handling this, we can change it.